### PR TITLE
Revert web3 py

### PR DESCRIPTION
**Pull request by pirapira**
This reverts #1153.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ dist: xenial
 sudo: true
 language: python
 python:
-  - '3.7'
+  - '3.6'
```
Comment (palango) [inline .travis.yml]: Should we leave this?

File: requirements-dev.txt [hunk 2]
```diff
@@ -13,9 +13,11 @@ flake8-tuple>=0.2.13
 bump2version
 hypothesis==3.44.14
 requests
-mypy==0.720
+mypy==0.670
```
Comment (palango) [inline requirements-dev.txt]: might also be nice to keep this up to date

Comment (LefterisJP) [reply requirements-dev.txt]: +1 ... we also use this mypy version in the client.

File: .travis.yml (hunk 1 above)
Comment (pirapira) [reply .travis.yml]: Yes, changed to 3.7.  I also changed another 3.6 to 3.7.

File: requirements-dev.txt (hunk 2 above)
Comment (pirapira) [reply requirements-dev.txt]: Kept the newer `mypy`.

**Pull request merged**